### PR TITLE
Fixed CPython failing to build from source on Cygwin

### PR DESCRIPTION
**Pull request by TheShermanTanker**
I was not able to find a corresponding bug logged anywhere, unfortunately. It appears that this issue has been under-reported

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept this contribution by verifying everyone involved has signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

## Recognized GitHub username

We couldn't find a [bugs.python.org](https://bugs.python.org/) (b.p.o) account corresponding to the following GitHub usernames:

@TheShermanTanker

This might be simply due to a missing "GitHub Name" entry in one's b.p.o account settings. This is necessary for legal reasons before we can look at this contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.




You can [check yourself](https://check-python-cla.herokuapp.com/) to see if the CLA has been received.

Thanks again for the contribution, we look forward to reviewing it!

File: Modules/Setup [hunk 1]
```diff
@@ -359,7 +359,7 @@ xxsubtype xxsubtype.c
 # Uncommenting the following line tells makesetup that all following modules
 # are not built (see above for more detail).
 #
-#*disabled*
+*disabled*
```
Comment (pablogsal) [inline Modules/Setup:362]: Hummm.... this looks incorrect. Why is this being changed?

File: Parser/pegen.c [hunk 2]
```diff
@@ -1,5 +1,6 @@
 #include <Python.h>
 #include <errcode.h>
+#include <errno.h>
```
Comment (pablogsal) [inline Parser/pegen.c:3]: Why is this change needed?

**Comment by pablogsal** [comment]
> I was not able to find a corresponding bug logged anywhere, unfortunately. It appears that this issue has been under-reported

Please, open an issue on bpo so we can also discuss the problem there.

File: Modules/Setup (hunk 1 above)
Comment (TheShermanTanker) [reply Modules/Setup:362]: My apologies, it appears that I left it there by mistake

File: Parser/pegen.c (hunk 2 above)
Comment (TheShermanTanker) [reply Parser/pegen.c:3]: > 
> 
> > I was not able to find a corresponding bug logged anywhere, unfortunately. It appears that this issue has been under-reported
> 
> Please, open an issue on bpo so we can also discuss the problem there.

Got it, will close PR and open issue on bpo, I'm confident the python team would be much more well equipped to handle this than I am (It's been driving me insane so far)

**Comment by TheShermanTanker** [comment]
> 
> 
> > I was not able to find a corresponding bug logged anywhere, unfortunately. It appears that this issue has been under-reported
> 
> Please, open an issue on bpo so we can also discuss the problem there.

Got it, will close PR and open issue on bpo, I'm confident the python team would be much more well equipped to handle this than I am (It's been driving me insane so far)
